### ISSUE TITLE
Install PF_RING for CentOS ISO

### ISSUE THREAD

**Issue by laoyang103** [ntop/PF_RING#425]
Hi, @cardigliano 

I download pfring-7.3.0-2206.x86_64.rpm install for CentOS, I install it after system installed, no error.
I want to install it with install system(add rpm package for ISO), got follow error, I know this is not your problem, could you tell me what command run when install pfring, so I can run it on the last step of system install

> Installing pfring-7.3.0-2206.x86_64
warning: pfring-7.3.0-2206.x86_64: Header V4 RSA/SHA1 Signature, key ID d1eb60be: NOKEY
/usr/local/bin/pf_ringctl: line 24: cut: command not found
/usr/local/bin/pf_ringctl: line 40: head: command not found
/usr/local/bin/pf_ringctl: line 40: tr: command not found
/usr/local/bin/pf_ringctl: line 40: cut: command not found
/usr/local/bin/pf_ringctl: line 616: /bin/date: No such file or directory
/usr/local/bin/pf_ringctl: line 82: lsmod: command not found
/usr/local/bin/pf_ringctl: line 82: wc: command not found
/usr/local/bin/pf_ringctl: line 82: [: -le: unary operator expected
/usr/local/bin/clusterctl: line 266: logger: command not found
/usr/local/bin/pf_ringctl: line 503: sleep: command not found
/usr/local/bin/pf_ringctl: line 336: /sbin/lsmod: No such file or directory
/usr/local/bin/pf_ringctl: line 336: wc: command not found
/usr/local/bin/pf_ringctl: line 336: [: -eq: unary operator expected
/usr/local/bin/pf_ringctl: line 336: /sbin/lsmod: No such file or directory
/usr/local/bin/pf_ringctl: line 336: wc: command not found
/usr/local/bin/pf_ringctl: line 336: [: -eq: unary operator expected
/usr/local/bin/pf_ringctl: line 336: /sbin/lsmod: No such file or directory
/usr/local/bin/pf_ringctl: line 336: wc: command not found
/usr/local/bin/pf_ringctl: line 336: [: -eq: unary operator expected
/usr/local/bin/pf_ringctl: line 336: /sbin/lsmod: No such file or directory
/usr/local/bin/pf_ringctl: line 336: wc: command not found
/usr/local/bin/pf_ringctl: line 336: [: -eq: unary operator expected
/usr/local/bin/pf_ringctl: line 336: /sbin/lsmod: No such file or directory
/usr/local/bin/pf_ringctl: line 336: wc: command not found
/usr/local/bin/pf_ringctl: line 336: [: -eq: unary operator expected
/usr/local/bin/pf_ringctl: line 190: cat: command not found
/usr/local/bin/pf_ringctl: line 190: cut: command not found
/usr/local/bin/pf_ringctl: line 190: tr: command not found
/usr/local/bin/pf_ringctl: line 508: wc: command not found
/usr/local/bin/pf_ringctl: line 509: [: -gt: unary operator expected
/usr/local/bin/pf_ringctl: line 593: mkdir: command not found
/usr/local/bin/pf_ringctl: line 596: touch: command not found
/usr/local/bin/pf_ringctl: line 599: touch: command not found
/usr/local/bin/pf_ringctl: line 602: touch: command not found
/usr/local/bin/pf_ringctl: line 603: /etc/pf_ring/interfaces.conf: No such file or directory
/usr/local/bin/pf_ringctl: line 604: /etc/pf_ring/interfaces.conf: No such file or directory
/usr/local/bin/pf_ringctl: line 609: mkdir: command not found
/usr/local/bin/pf_ringctl: line 609: mkdir: command not found
/usr/local/bin/pf_ringctl: line 609: mkdir: command not found
/usr/local/bin/pf_ringctl: line 609: mkdir: command not found
/usr/local/bin/pf_ringctl: line 609: mkdir: command not found
/usr/local/bin/pf_ringctl: line 652: /bin/date: No such file or directory
/usr/local/bin/clusterctl: line 266: logger: command not found
/usr/local/bin/clusterctl: line 295: sleep: command not found

**Comment by cardigliano** [comment]
@laoyang103 the post installation script is calling "/bin/systemctl restart pf_ring", which is calling "/usr/local/bin/pf_ringctl start". I am not sure I got your question, could you rephrase please?

**Comment by laoyang103** [comment]
@cardigliano OK, I know.  my question is when run postinst some command not found, for example 

> /usr/local/bin/pf_ringctl: line 602: touch: command not found

**Comment by cardigliano** [comment]
@laoyang103 in some case we need to add the absolute path, in other cases I see that even commands with the absolute path are not found (e.g. /sbin/lsmod). What centos are you running?

**Comment by laoyang103** [comment]
@cardigliano I use centos 6.10, it doesn't matter, I have run after all software installed.
